pkg/unvanquished_src.dpkdir/ui/shared/basics.rcss

```diff
@@ -62,6 +62,11 @@ indent {
 	margin-left: 1.5em;
 }
 
+web {
+	color: #136d15; /* Green */
```

Comment:
No it's not supposed to look like a link, because indeed you can't click on it. I just wanted to make these more obvious to spot in the page, but maybe that may be confusing if people believes it's a link.

Reply:
I've set it white for now. But even white can be confused with a link unless we color the main menu links like other in-game links.